test/model/create.test.ts

```diff
@@ -46,6 +46,34 @@ test('creates a new entity with an array property', () => {
   expect(exactUser).toHaveProperty('arrayProp', [1, 2, 3])
 })
 
+test('creates a new entity with an array property with array of objects assigned', () => {
+  const db = factory({
+    user: {
+      id: primaryKey(datatype.uuid),
+      arrayProp: Array,
```

Comment:
Let's come up with a property name that's closer to how people would model real-world data with arrays. This would make our test shareable as usage examples, which is a great feat for a test. 

```suggestion
      notes: Array,
```

Perhaps, let's consider `user.notes`, where each note is, basically, a list of `key/value` pairs (as you have it now)?

Reply:
Make sense! I've adjusted the example as suggested in https://github.com/mswjs/data/pull/169/commits/a29fa1c69d10a6870612601aa0d7320e63e97f63

Reply:
Fantastic! Thank you for tackling this.